test/integration/app2/inspec/app2_spec.rb

```diff
@@ -69,3 +64,18 @@
     its('stdout') { should match %r{\(ALL\) NOPASSWD: /usr/bin/systemctl restart #{app}} }
   end
 end
+
+describe file('/var/lib/formsender/Dockerfile') do
+  it { should exist }
+end
+
+describe docker.images.where { tag == 'latest' } do
+  it { should exist }
+end
+
+describe docker_container('support.osuosl.org') do
+  it { should exist }
+  it { should be_running }
+  its('image') { should eq 'formsender:latest' }
+  its('ports') { should eq '0.0.0.0:8086->5000/tcp' }
+end
```

Comment:
Can you add an `http` test which confirms that the container has been deployed properly?